_config/security.yml

```diff
@@ -1,4 +1,33 @@
-SilverStripe\Security\MemberLoginForm:
+---
+Name: coresecurity
+---
```

Comment:
Can we please add a Name: coresecurity block to the header, so that users can extend this block using ordered overrides?